### ISSUE TITLE
Make menu bar more obvious

### ISSUE THREAD

**Issue by AnnaSyme** [melbournebioinformatics/MelBioInf_docs#48]
On the main page - https://www.melbournebioinformatics.org.au/tutorials/ - , the menu bar is in the top left corner (three lines, but not named). For some people this won't be obvious. See if there's a way to point to this / make more obvious.

**Comment by claresloggett** [comment]
This happens when the browser window is (supposedly) too narrow to comfortably fit both the menu sidebar and the page content. If you widen the browser window, the sidebar should reappear. It's actually an intended feature - I think of the material theme, rather than of mkdocs itself. But I agree it is not obvious. I spent some time looking at fixes when I rebuilt the site, but didn't find a solution I liked.

We can set `tabs: true` in `mkdocs.yml` to get tabs along the top of the page instead of a sidebar. However I don't think it's possible to have tabs *as well as* a sidebar, which I would prefer. 

We could:

* switch to tabs instead of a sidebar menu
* ask at mkdocs-material whether there's a way to have both menus, and if not make it a feature request, or try to do it ourselves as a PR if they'd accept it
* work out how to tweak the browser width at which the menu-hiding occurs, if we think it's happening at too wide a width
